pkg/actor/decommission.go

```diff
@@ -57,7 +57,13 @@ func (d decommission) Handles(conds []api.ClusterCondition) bool {
 }
 
 func (d decommission) Act(ctx context.Context, cluster *resource.Cluster) error {
+
 	log := d.log.WithValues("CrdbCluster", cluster.ObjectKey())
+
+	if !utilfeature.DefaultMutableFeatureGate.Enabled(features.Decommission) {
```

Comment:
We missed the feature gate for decommision.  I moved them into the actors so that building the slice is easier.

Reply:
@chrislovecnm The decommission flag  it is checked in the Handle method. If Handle returns true than Act. No need for this condition on Act.

Reply:
Awesome! I will change mine to match